app/apps/server/bridges/users.js

```diff
@@ -1,4 +1,9 @@
+import { Meteor } from 'meteor/meteor';
+
+import { saveUser } from '../../../lib/server/functions/saveUser';
 import { Users } from '../../../models/server';
```

Comment:
You can import `Roles` model from here
```suggestion
import { Users, Roles } from '../../../models/server';
```